src/webclients/components/sdfv/utils/attributes_table.ts

```diff
@@ -1603,7 +1612,7 @@ export function appendDataDescriptorTable(
             }).appendTo(attrTable);
             attributeTablePutEntry(
                 descriptor, val, attrMeta, descriptors, undefined,
-                undefined, row, true, true, true, updateNameListener,
+                undefined, row, true, true, true, false, updateNameListener,
```

Comment:
I'm guessing that all these `false`s are "the property was not modified". Right?